docs/user-guide/KubernetesCurator.rst

```diff
@@ -0,0 +1,382 @@
+    # Change foobar to the directory or file you wish to upload.
+    ( cd $LOCAL_WORKSPACE; tar cf - big_english | kubectl exec -i nemo-workspace-busybox -- tar xf - -C /nemo-workspace )
+
+.. note::
+    See :ref:`data-curator-download` for an example of how to download local data that can be uploaded to the PVC
+    with the above instruction.
+
+Create a Dask Cluster
+---------------------
+
+Use the ``create_dask_cluster.py`` to create a CPU or GPU dask cluster.
+
+.. note::
+    If you are creating another Dask cluster with the same ``--name <name>``, first delete it via::
+
+        kubectl delete daskcluster <name>
+
+.. code-block:: bash
+
+    # Creates a CPU Dask cluster with 1 worker
+    python create_dask_cluster.py \
+        --name rapids-dask \
+        --n_workers 1 \
+        --image nvcr.io/nvidian/bignlp-train:nemofw-nightly \
+        --image_pull_secret ngc-registry \
+        --pvcs nemo-workspace:/nemo-workspace
+
+    #╭───────────────────── Creating KubeCluster 'rapids-dask' ─────────────────────╮
+    #│                                                                              │
+    #│   DaskCluster                                                      Running   │
+    #│   Scheduler Pod                                                    Running   │
+    #│   Scheduler Service                                                Created   │
+    #│   Default Worker Group                                             Created   │
+    #│                                                                              │
+    #│ ⠧ Getting dashboard URL                                                      │
+    #╰──────────────────────────────────────────────────────────────────────────────╯
+    #cluster = KubeCluster(rapids-dask, 'tcp://localhost:61757', workers=2, threads=510, memory=3.94 TiB)
+
+    # Creates a GPU Dask cluster with 2 workers with 1 GPU each
+    python create_dask_cluster.py \
+        --name rapids-dask \
+        --n_workers 2 \
+        --n_gpus_per_worker 1 \
+        --image nvcr.io/nvidian/bignlp-train:nemofw-nightly \
+        --image_pull_secret ngc-registry \
+        --pvcs nemo-workspace:/nemo-workspace
+
+After creating a cluster, you should be able to proceed after confirming the scheduler and the workers are all ``Running``:
+
+.. code-block:: bash
+
+    # Set DASK_CLUSTER_NAME to the value of --name
+    DASK_CLUSTER_NAME=rapids-dask
+    kubectl get pods -l "dask.org/cluster-name=$DASK_CLUSTER_NAME"
+
+    # NAME                                                     READY   STATUS    RESTARTS      AGE
+    # rapids-dask-default-worker-587238cf2c-7d685f4d75-k6rnq   1/1     Running   0             57m
+    # rapids-dask-default-worker-f8ff963886-5577fff76b-qmvcd   1/1     Running   3 (52m ago)   57m
+    # rapids-dask-scheduler-654799869d-9bw4z                   1/1     Running   0             57m
+
+.. tab-set::
+
+    .. tab-item:: Running Existing Module
```

Comment:
These tab items don't really stand out on GitHub. They might on the NeMo user guide, but I'd prefer if we could get these docs to look good on both the user guide and GitHub. Can you change this so `Running Existing Module` is more noticeable (like a subheading or something)?